RecFCCeeCalorimeter/src/components/CalibrateBenchmarkMethod.cpp

```diff
@@ -0,0 +1,279 @@
+#include "CalibrateBenchmarkMethod.h"
+
+// Key4HEP
+#include "k4Interface/IGeoSvc.h"
+
+// datamodel
+#include "edm4hep/CalorimeterHit.h"
+#include "edm4hep/CalorimeterHitCollection.h"
+
+// DD4hep
+#include "DD4hep/Detector.h"
+#include "DD4hep/Readout.h"
+
+#include "GaudiKernel/ITHistSvc.h"
+
+#include "TH1F.h"
+#include "TVector3.h"
+#include "TMath.h"
+#include "Math/Minimizer.h"
+#include "Math/Factory.h"
+#include "Math/Functor.h"
+#include "TRandom2.h"
+#include "TError.h"
+
```

Comment:
Where are those declared?

Reply:
Ahh, I see --- those should be members of the algorithm class then.

Reply:
it is fixed now